templates/registration/login.html

```diff
@@ -91,7 +91,8 @@
             <div id="main-menu" class="cfix login-menu" style="background: {% settings_value 'login_background' %}">
                 <div class="vw-menu pure-menu-horizontal container cfix">
                         <a id="logo-link" href="{% settings_value 'logo_link' %}" class="pure-menu-heading left">
-                            <span><img id="ona-logo" src="{% settings_value 'login_logo' %}" alt="ONA" /></span>
+                            <span><img id="ona-logo" src="{% settings_value 'login_logo' %}" 
+                                alt="ONA" onerror="this.onerror=null;this.src='/static/who-logo.png';"/></span>
```

Comment:
let's remove the `onerror` attribute here

Reply:
The image extension was not set correctly...and the value set dynamically from elsewhere. I wasn't sure where to find it. Thats when I thought of the same...

Reply:
it's set dynamically from the deployment configurations